packages/@react-spectrum/menu/docs/Menu.mdx

```diff
@@ -379,6 +460,13 @@ import {Content} from '@react-spectrum/view';
   }
 }} links={contextualHelpTriggerTypes.links} />
 
+### SubmenuTrigger Props
+<PropTable component={{
```

Comment:
this is unusual, what was happening to the usual way of doing this?

Reply:
ah, should we split the type? i think i wound up doing that for contextual help trigger?

Reply:
thats the thing, I did split the types: https://github.com/adobe/react-spectrum/blob/98869934e42833a6c0dc626ab23482a6d401ef03/packages/%40react-spectrum/menu/src/SubmenuTrigger.tsx#L24-L34 
https://github.com/adobe/react-spectrum/blob/98869934e42833a6c0dc626ab23482a6d401ef03/packages/%40react-spectrum/menu/src/SubmenuTrigger.tsx#L152

so I was expecting 
```
<PropTable component={docs.exports.SubmenuTrigger} links={docs.links} />
```
to have the proper props table, but it still included the `targetKey` prop

Reply:
What about
```
<PropTable component={docs.exports.SpectrumSubmenuTriggerProps} links={docs.links} />
```